### PR TITLE
Added Chimera LS and Chimera Ergo

### PR DESCRIPTION
**Pull request by GlenPickle**
I'm adding two new keyboards to QMK, the Chimera Ergo and Chimera LS. Both are wireless split keyboards with open source gerber files

### CODE REVIEW

File: keyboards/chimera_ortho/chimera_ortho.h [hunk 1]
```diff
@@ -63,4 +63,17 @@
 	{ KC_NO,      KC_##k28, KC_##k14, KC_##k00, KC_##k45,      KC_##k48, KC_##k13, KC_##k27, KC_##k44, KC_NO }, \
 }
 
+#define KC_KEYMAP( \
```
Comment (jackhumbert) [inline keyboards/chimera_ortho/chimera_ortho.h]: I'm not sure how I missed this before, but it looks like you have these backwards - the `KC_KEYMAP` should prepend `KC_` where the `KEYMAP` should just pass them straight through :)

**Comment by GlenPickle** [comment]
I fixed the KEYMAP/KC_KEYMAP mixup. I still need to fix @DanielGGordon keymap at some point so that the build won't fail

**Comment by drashna** [comment]
@GlenPickle turning on tapdance or wrapping the tapdance code in `ifdef`'s should/will fix the issue.

**Comment by jackhumbert** [comment]
Awesome - thanks! We can deal with the failing later :)

**Pull request merged**